bpf/tests/tc_nodeport_l3_dev.c

```diff
@@ -63,6 +62,8 @@ static volatile const __u8 *node_mac = mac_two;
 
 #include "bpf_host.c"
 
+ASSIGN_CONFIG(__u32, host_secctx_from_ipcache, 1)
+
```

Comment:
1. Add the following after `#include "bpf_host"`

```suggestion
ASSIGN_CONFIG(__u32, interface_ifindex, WG_IFINDEX)
```
2. delete `#define THIS_INTERFACE_IFINDEX WG_IFINDEX` at the top of the file.
3. Repeat step (1,2) for `wireguard_metrics.c`

Reply:
@smagnani96 No worries, it's an easy fix. This will happen quite frequently over the coming months as we port over overridable config macros from the preprocessor to CONFIG(). All good.